### ISSUE TITLE
Arming broken

### ISSUE THREAD

**Issue by ryanmilvenan** [schwark/pyalarmcom#1]
Status and disarm seem to work fine, but arming stay or away gets a 422. Is it still working for you?

**Comment by ryanmilvenan** [comment]
I changed your action for silent from store_false to store_true and it works again. This seems to sorta revert your most recent change.

**Comment by dswee999** [comment]
@ryanmilvenan are you implanting this through Home Assistant or Home Bridge? I'm not able to get it to work. Any help would be appreciated. Thanks

**Comment by ryanmilvenan** [comment]
Yeah, Home Bridge. if you check out [this issue](https://github.com/bryan-bartow/homebridge-alarm.com/issues/71), I'll write a little bit more about it there.

**Comment by timmerk** [comment]
Arming stay (and all other functions) works for me as of today, Jan 21, 2019.
